src/App.js

```diff
@@ -1,9 +1,13 @@
-import './App.css';
+import './app.css';
```

Comment:
Great job implementing the project 👍 , I am unable to run the app because of the error in the import of the CSS file.  On your file directory, it is App.css and not app.css. Kindly check it out. 🥺 

<img width="1470" alt="image" src="https://github.com/ShaliniNayan/math-magicians/assets/73854999/41398128-8344-4b36-b507-21811f8d1f27">